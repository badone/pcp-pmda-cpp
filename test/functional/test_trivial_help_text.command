TEMP_DIR=`mktemp -d -t test_trivial_text-XXXXXXXXXX`
./pmdatrivial --export-all "$TEMP_DIR"
if [ $? -ne 0 ]; then exit; fi
dbpmda -e -n "$TEMP_DIR/root" <<EOF
open pipe pmdatrivial
text trivial.time
close
EOF
rm -rf "$TEMP_DIR"
