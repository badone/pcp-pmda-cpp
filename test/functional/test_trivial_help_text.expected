open pipe pmdatrivial
Start pmdatrivial PMDA: ./pmdatrivial
text trivial.time
PMID: 250.0.0
[The time in seconds since 1 Jan 1970]
The time in seconds since the 1st of January, 1970.
